### PR TITLE
Run the psmash linter.

### PR DESCRIPTION
**Pull request by rbarlow**

### CODE REVIEW

**Comment by Ichimonji10** [comment]
Is the idea here that a developer can write tests in parallel with test code? Cool! If that's the case, consider adding in `make test && ` before the `make lint` part. `make test` causes Pulp Smash to run its own internal unit tests. It's extremely fast, and doesn't trigger any interactions with your Pulp system.

File: playpen/ansible/roles/dev/files/bashrc [hunk 1]
```diff
@@ -75,6 +75,7 @@ _paction() {
 psmash() {
     workon pulp-smash;
     prestart;
+    make lint || return 1;
```
Comment (seandst) [inline playpen/ansible/roles/dev/files/bashrc]: returning 1 here prevents  the later call to deactivate

I recommend something like

```bash
if make lint
then
    python -m unittest2 discover ~/devel/pulp-smash/pulp_smash;
fi
deactivate
```

**Comment by seandst** [comment]
> make test && before the make lint

I think `make lint test` would probably be a more readable option, and I think I agree that if we're linting the pulp smash tests, we might as well run them, too.

**Comment by Ichimonji10** [comment]
> I think `make lint test` would probably be a more readable option

I somehow did not know that you could specify multiple make targets with a single invocation of `make`. Thanks.

**Comment by seandst** [comment]
Not only that, but make doesn't continue after a failed target (by default, at least), so the behavior is identical to `make lint && make test`

File: playpen/ansible/roles/dev/files/bashrc (hunk 1 above)
Comment (rbarlow) [inline playpen/ansible/roles/dev/files/bashrc]: Oh I like that, much nicer. My bash foo is so terrible.

**Comment by rbarlow** [comment]
I have updated this PR to use make lint test with an if statement.

**Comment by Ichimonji10** [comment]
:+1:

**Comment by rbarlow** [comment]
ok test

**Comment by rbarlow** [comment]
I am confident that the failing test here is not due to this pull request.

**Pull request merged**